skills/csharp/experimental/itsmskill/Dialogs/CreateTicketDialog.cs

```diff
@@ -83,7 +83,7 @@ public class CreateTicketDialog : SkillDialogBase
 
         protected override async Task<DialogTurnResult> OnBeginDialogAsync(DialogContext dc, object options, CancellationToken cancellationToken = default)
         {
-            if (dc.Context.Activity.ChannelId.Contains("msteams"))
+            if (dc.Context.Activity.ChannelId.Contains(Channels.Msteams))
```

Comment:
Use `==` here? channelId is set by channel and it is supposed to be one of these constants.
They all use `==` in sdk

Reply:
we should use == instead of contains

Reply:
changed to ==